ledger-core/v2/vanilla/throw/wraps.go

```diff
@@ -102,10 +98,11 @@ func (v stackWrap) LogString() string {
 }
 
 func (v stackWrap) Error() string {
-	if v.st == nil || v.stDeepest != nil {
-		return v.LogString()
-	}
-	return joinStack(v.LogString(), v.st)
+	//if v.stDeepest != nil {
```

Comment:
Remove it pls, if it's not needed

Reply:
we didn't decide yet - should error.Error() return a stack trace or not